jormungandr/src/blockchain/chain.rs

```diff
@@ -332,15 +338,15 @@ impl Blockchain {
     pub fn get_ref(
         &self,
         header_hash: HeaderHash,
-    ) -> impl Future<Item = Option<Arc<Ref>>, Error = Error> {
+    ) -> impl Future01<Item = Option<Arc<Ref>>, Error = Error> {
```

Comment:
Hmm, so you did not convert all `Blockchain` methods at once...

Reply:
too many functions to change at once, this makes a lot of churn already, better to spread it along